insights/parsers/dse_ldif.py

```diff
@@ -0,0 +1,194 @@
+"""
+RHDSConfig - file ``/etc/dirsrv/slapd-<>/dse.ldif``
+===================================================
+
+The ``dseLdif`` module provides parsing for the
+``dse.ldif`` file.  The ``dseLdifConfig`` class
+implements the parsing and provides a ``list`` of all
+configuration lines present in the file.
+
+Sample input is provided in the *Examples*.
+
+Examples:
+    >>> dse_ldif_input = '''
+    ...
+    ... dn:
+    ... objectClass: top
+    ... aci: (targetattr != "aci")(version 3.0; aci "rootdse anon read access"; allow(
+    ...  read,search,compare) userdn="ldap:///anyone";)
+    ... creatorsName: cn=server,cn=plugins,cn=config
+    ... modifiersName: cn=server,cn=plugins,cn=config
+    ... createTimestamp: 20151021124922Z
+    ... modifyTimestamp: 20151021124922Z
+    >>> from insights.tests import context_wrap
+    >>> shared = {dseLdifConfig: dseLdifConfig(context_wrap(dse_ldif_input))}
+    >>> dse_ldif_config = shared[dseLdifConfig]
+    >>> 'objectClass' in dse_ldif_config
+    True
+    >>> 'creatorsName' in dse_ldif_config
+    True
+    >>> dse_ldif_config.get_line('creatorsName')
+    'creatorsName: cn=server,cn=plugins,cn=config'
+    >>> dse_ldif_config.get_values('modifyTimestamp')
+    ['20151021124922Z']
+    >>> dse_ldif_config.last('createTimestamp')
+    '20151021124922Z'    
+"""
```

Comment:
@bfahr Thanks for response bob. Can you provide reference to parser(instead of combiner). Also, parsing a configuration file?

Reply:
@bfahr Also It would be easy for me if you provide some matching configuration file. ie key:value pairs separated by colon. Example:
objectClass: top

Thank you very much in Advance!

Reply:
@bfahr Any updates?

Reply:
@amitkumar50 you can see another [example docstring](https://github.com/RedHatInsights/insights-core/blob/6905ca42d30cdee5c9e9c97ce1fa278a47779c3b/insights/parsers/rhsm_releasever.py#L28)  can be seen here, and the associated [tests](https://github.com/RedHatInsights/insights-core/blob/6905ca42d30cdee5c9e9c97ce1fa278a47779c3b/insights/parsers/tests/test_rhsm_releasever.py#L45) here.

Reply:
@bfahr Thank you very much, Changed docstring